actions/src/Upload.re

```diff
@@ -0,0 +1,23 @@
+open Globals;
+
+// Mutation to change sync status + update endTime.
+module SyncItemUpdate = [%graphql
+  {|
+    mutation SyncItemMutation($Id: Int!, $endTime: Int!, $status: String!) {
+    update_bridge_sync_by_pk(pk_columns: {id: $Id}, _set: {end_time: $endTime, status: $status}) {
+        end_time
+        status
+    }
+  }
+|}
+];
+
+// Will take in the skaleEndPoint (so we know what data to upload)
+// The arweave endpoint so we know where to upload it
+// What else? Should use users arweave keys for this action
+let uploadChunkToArweave = () => {
```

Comment:
Cool, we will need to give this function parameters, but that is easy to sort out in the future.